src/conditional-compilation.md

```diff
@@ -25,7 +25,7 @@
 *Conditionally compiled source code* is source code that may or may not be
 considered a part of the source code depending on certain conditions. <!-- This
 definition is sort of vacuous --> Source code can be conditionally compiled
-using [attributes], [`cfg`] and [`cfg_attr`], and the built-in [`cfg` macro].
```

Comment:
This one is actually semantically a list of two items. Adding the comma there would be a comma splice as well. It's meant to be read as "using attributes (cfg and cf_attr) and the built-in cfg macro".

Reply:
Are cfg and cf_attr the only attributes that can do conditional compilation? If so, I think that it would read well like:

"using the attributes cfg and cf_attr and the built-in cfg macro."

I read it (without the "the") as a non-exhaustive list of example of attributes that enable conditional compilation. I am definitely not trying to nittle, I am just trying to make sure that I understand the sentence. 

Thank you!

Reply:
The `the` there definitely helps. I also felt like something was missing, but couldn't think of what specifically.

Reply:
Perfect! Thanks!